lib/ts/controllers/s-expandable-control.ts

```diff
@@ -79,13 +79,13 @@
         };
 
 
-        get controlledCollapsible() {
+        get controlledCollapsibles() {
             const attr = this.element.getAttribute("aria-controls");
             if (!attr) {
                 throw "couldn't find controls"
             }
-            const result = document.getElementById(attr);
-            if (!result) {
+            const result = attr.split(/\s+/g).map(s => document.getElementById(s)).filter((e): e is HTMLElement => !!e);
+            if (result.length == 0) {
```

Comment:
optional: very minor code cleanup. Also, nice use of type guards here!

```suggestion
            const result = attr.split(/\s+/g)
                .map(s => document.getElementById(s))
                .filter((e): e is HTMLElement => !!e);
            if (!result.length) {
```

Reply:
Thanks! All credit goes to Sublime's TypeScript plugin telling me "object is possibly null" in places until I figured it out what type guards even were 😅